### PR TITLE
Additional checks for var existence in Admin Features controller

### LINKED ISSUE #23423: [BO] - Warning displayed on Feature value page
#### Describe the bug

A alert danger is displayed when trying to access Feature values page

#### Expected behavior

No exception 

#### Steps to Reproduce

1. Go to Catalog Attributes and features
2. Click on Features tab
3. View a features to see values

**Screenshots**

image

#### Additional information

* PrestaShop version: develop branch
* PHP version: 7.4 (works on 7.3)

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | compat with PHP 7.4
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23423
| How to test?      | Follow #23423
| Possible impacts? | n/a
| Sponsor company | impSolutions

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23425)

### CODE REVIEW

File: controllers/admin/AdminFeaturesController.php [hunk 1]
```diff
@@ -292,7 +292,7 @@ public function initToolbarTitle()
                 break;
 
             case 'view':
-                $bread_extended[] = $this->feature_name[$this->context->employee->id_lang];
+                $bread_extended[] = $this->feature_name[$this->context->employee->id_lang] ?? null;
```
Comment (sowbiba) [inline controllers/admin/AdminFeaturesController.php:295]: If the value is null, you will call $this->addMetaTitle(null);
Do you know what are the consequences ?

Comment (kpodemski) [reply controllers/admin/AdminFeaturesController.php:295]: yes, none, as there's a check `is_array` inside this method and `$bread_extended` is created a bit sooner with rest of the breadcrumbs displayed, I tested it before and after change and haven't noticed any differences 🤔

**Review by sowbiba — APPROVED** [review]

**Review by matks — APPROVED** [review]
PHP7.4 compatibility is making PS code overall better 😄 without php7.4 strictness we would not have seen this bug. Great job !

**Comment by SD1982** [comment]
LGTM !!
Thanks @kpodemski !!

**Comment by Progi1984** [comment]
Thanks @kpodemski & @SD1982

**Pull request merged**